frontends/web/public/btcdirect/fiat-to-coin.html

```diff
@@ -10,104 +10,123 @@
+              : 'https://cdn-sandbox.btcdirect.eu/fiat-to-coin/fiat-to-coin.css'
+            ),
+            rel: 'stylesheet',
+          })
+        );
 
-    // add the btcdirect CSS
-    document.head.appendChild(
-      Object.assign(document.createElement('link'), {
-        href: (
+        // add the btcdirect script
+        (function (btc, d, i, r, e, c, t) {
+          btc[r] = btc[r] || function () {
+            (btc[r].q = btc[r].q || []).push(arguments)
+          };
+          c = d.createElement(i);
+          c.id = r; c.src = e; c.async = true;
+          c.type = 'module'; c.dataset.btcdirect = '';
+          t = d.getElementsByTagName(i)[0];
+          t.parentNode.insertBefore(c, t);
+        })(window, document, 'script', 'btcdirect',
           mode === 'production'
-          ? 'https://cdn.btcdirect.eu/fiat-to-coin/fiat-to-coin.css'
-          : 'https://cdn-sandbox.btcdirect.eu/fiat-to-coin/fiat-to-coin.css'
-        ),
-        rel: 'stylesheet',
-      })
-    );
-
-    // add the btcdirect script
-    (function (btc, d, i, r, e, c, t) {
-      btc[r] = btc[r] || function () {
-        (btc[r].q = btc[r].q || []).push(arguments)
-      };
-      c = d.createElement(i);
-      c.id = r; c.src = e; c.async = true;
-      c.type = 'module'; c.dataset.btcdirect = '';
-      t = d.getElementsByTagName(i)[0];
-      t.parentNode.insertBefore(c, t);
-    })(window, document, 'script', 'btcdirect',
-      mode === 'production'
-      ? 'https://cdn.btcdirect.eu/fiat-to-coin/fiat-to-coin.js'
-      : 'https://cdn-sandbox.btcdirect.eu/fiat-to-coin/fiat-to-coin.js'
-    );
-
-    btcdirect('init', {
-      token: apiKey,
-      debug: mode === 'debug',
-      locale: window.frameElement?.dataset.locale || 'en-US',
-      theme: window.frameElement?.dataset.theme || 'light',
-    });
-
-    // fiat to coin order
-    btcdirect('wallet-addresses', {
-      addresses: {
-        address,
-        currency,
-        id: 'BitBox',
-        walletName: 'BitBox'
-      }
-    });
-
-    btcdirect('set-parameters',
-      mode === 'production' ? {
-        baseCurrency: currency,
-        fixedCurrency: true,
-        quoteCurrency,
-        // paymentMethod: any of 'bancontact', 'bankTransfer', 'creditCard', 'giropay', 'iDeal', 'sofort', 'applepay'
-        showWalletAddress: false,
-      } : {
-        baseCurrency: currency,
-        fixedCurrency: true,
-        paymentMethod: 'sofort', // sandbox currently only supports sofort payment method
-        quoteCurrency,
-        showWalletAddress: false,
+          ? 'https://cdn.btcdirect.eu/fiat-to-coin/fiat-to-coin.js'
+          : 'https://cdn-sandbox.btcdirect.eu/fiat-to-coin/fiat-to-coin.js'
+        );
+
+        btcdirect('init', {
+          token: apiKey,
+          debug: mode === 'debug',
+          locale: window.frameElement?.dataset.locale || 'en-GB',
+          theme: window.frameElement?.dataset.theme || 'light',
```

Comment:
TODO: take this from onMessage "configuration" / event.data

Reply:
rebased this branch and fixed in https://github.com/BitBoxSwiss/bitbox-wallet-app/pull/3152